### PR TITLE
Add `scanning` section into navbar.

### PR DESCRIPTION
**Pull request by AI-Mozi**
#56

### CODE REVIEW

File: public/stylesheets/app.css [hunk 1]
```diff
@@ -169,3 +169,9 @@ body.dark-mode main a {
 	right: 0.2rem;
 	top: 0;
 }
+
+.navbar-dropdown {
+	background-color: black;
+	padding-top: 0;
+	border-top: 0;
```
Comment (AI-Mozi) [inline public/stylesheets/app.css:176]: With `border-top` set the width difference between `scanning` item and dropdown section is annoying.
Without it being set, it's hard to notice the difference (imo).
If you want it to be fixed anyway, just let me know.

Comment (postmodern) [reply public/stylesheets/app.css:176]: I tested locally and it works fine. The font for the sub-menu seems a tad small, but we can fix that later.

File: views/layout.erb [hunk 2]
```diff
@@ -46,12 +46,20 @@
               <a href="/payloads" class="navbar-item">payloads</a>
               <a href="/exploits" class="navbar-item">exploits</a>
               <a href="/db" class="navbar-item">database</a>
-              <a href="/nmap" class="navbar-item">nmap</a>
-              <a href="/masscan" class="navbar-item">masscan</a>
               <a href="/import" class="navbar-item">import</a>
-              <a href="/spider" class="navbar-item">spider</a>
-              <a href="/recon" class="navbar-item">recon</a>
-              <a href="/vulns" class="navbar-item">vulns</a>
+
```
Comment (postmodern) [inline views/layout.erb:50]: I checked `git log -u` locally and git is saying there's extra whitespace on lines 50 and 62.

**Review by postmodern — CHANGES_REQUESTED** [review]
Tested locally and it works great! I did spot some extra whitespace that got left in `views/layout.erb`.

**Review by postmodern — APPROVED** [review]

**Pull request merged**